databricks_cli/pipelines/cli.py

```diff
@@ -62,9 +62,9 @@
 @provide_api_client
 def deploy_cli(api_client, spec_arg, spec, allow_duplicate_names, pipeline_id):
     """
-    Deploys a delta pipeline according to the pipeline specification. The pipeline spec is a
-    specification that explains how to run a Delta Pipeline on Databricks. All local libraries
-    referenced in the spec are uploaded to DBFS.
+    Deploys a pipeline according to the pipeline specification. The pipeline spec is a
+    JSON document that defines the required settings to run a Delta Live Tables pipelines
```

Comment:
```suggestion
    JSON document that defines the required settings to run a Delta Live Tables pipeline
```

This should be singular to match the the "run a".